api/src/test/java/org/pmiops/workbench/api/ClusterControllerTest.java

```diff
@@ -242,7 +294,96 @@ public void testListClusters() throws Exception {
   }
 
   @Test
-  public void testListClustersUnknownStatus() throws Exception {
+  public void testDeleteClustersInProject() {
+    List<ListClusterResponse> listClusterResponseList = ImmutableList.of(testFcClusterListResponse);
+    when(notebookService.listClustersByProjectAsAdmin(BILLING_PROJECT_ID))
+        .thenReturn(listClusterResponseList);
+
+    clusterController.deleteClustersInProject(
+        BILLING_PROJECT_ID,
+        new ListClusterDeleteRequest()
+            .clustersToDelete(ImmutableList.of(testFcCluster.getClusterName())));
+    verify(notebookService)
+        .deleteClusterAsAdmin(BILLING_PROJECT_ID, testFcCluster.getClusterName());
+    verify(clusterAuditor)
+        .fireDeleteClustersInProject(
+            BILLING_PROJECT_ID,
+            listClusterResponseList.stream()
+                .map(ListClusterResponse::getClusterName)
+                .collect(Collectors.toList()));
+  }
+
+  @Test
+  public void testDeleteClustersInProjectDeleteSome() {
+    List<ListClusterResponse> listClusterResponseList =
+        ImmutableList.of(testFcClusterListResponse, testFcClusterListResponse2);
+    List<String> clustersToDelete = ImmutableList.of(testFcCluster.getClusterName());
+    when(notebookService.listClustersByProjectAsAdmin(BILLING_PROJECT_ID))
+        .thenReturn(listClusterResponseList);
+
+    clusterController.deleteClustersInProject(
+        BILLING_PROJECT_ID, new ListClusterDeleteRequest().clustersToDelete(clustersToDelete));
+    verify(notebookService, times(clustersToDelete.size()))
+        .deleteClusterAsAdmin(BILLING_PROJECT_ID, testFcCluster.getClusterName());
+    verify(clusterAuditor, times(1))
+        .fireDeleteClustersInProject(BILLING_PROJECT_ID, clustersToDelete);
+  }
+
+  @Test
+  public void testDeleteClustersInProjectDeleteDoesNotAffectOtherProjects() {
+    List<ListClusterResponse> listClusterResponseList =
+        ImmutableList.of(testFcClusterListResponse, testFcClusterListResponse2);
+    List<String> clustersToDelete =
+        ImmutableList.of(testFcClusterDifferentProject.getClusterName());
+    when(notebookService.listClustersByProjectAsAdmin(BILLING_PROJECT_ID))
+        .thenReturn(listClusterResponseList);
+
+    clusterController.deleteClustersInProject(
+        BILLING_PROJECT_ID, new ListClusterDeleteRequest().clustersToDelete(clustersToDelete));
+    verify(notebookService, times(0))
```

Comment:
nit, optional: there's a `verifyZeroCalls()` (or similar) that can be used instead

Reply:
`verifyZeroInteractions()` is a mock level method, rather than a method level method (from what I could tell. Sticking with this.